### ISSUE TITLE
A11y: aria-labels

### ISSUE THREAD

**Issue by tbrockman** [tbrockman/prune#289]
**Describe the solution you'd like**
Input settings in popup are missing aria-labels, which is bad for accessibility. Fix this.
